### DISCUSSION TITLE
Any tips for debugging error DRT-0073 (No access point for ...)

### DISCUSSION THREAD

**Discussion by rbarzic** [The-OpenROAD-Project/OpenROAD-flow-scripts#1810] [Q&A]
Hi,

I'm trying to port ORFS to a TSMC process.  During the detailed route step, I get a lot of  DRT-0073 errors. Utilization is set to 20% at the beginning of the P&R flow and it is a pretty simple design. Has anyone some tips about what to look into ?
I have tried to run the pin_access command directly but the -verbose parameter does not seem to work as I don't get any more information

```
[ERROR DRT-0073] No access point for _3504_/A1.
[ERROR DRT-0073] No access point for _3046_/A2.
[INFO DRT-0077]   Complete 3000 pins.
[ERROR DRT-0073] No access point for _4260_/A.
[ERROR DRT-0073] No access point for _2948_/B1.
DRT-0073
while evaluating pin_access -verbose 2
```

**Comment by maliberty** [comment]
You can try the developer debugs to get a sense of what is going on.  Put

`detailed_route_debug -pa -pa_markers -pin _3504_:A1`

before you call detailed_route and run in the GUI.  It will stop at the pin and show you how it is trying to access it and any associated drc markers.  You click the continue button to see the various steps.  Hopefully that gives you an idea as to what is going on.

The other option is a support contract with precisioninno.com where we can help under NDA.

**Reply by rbarzic** [reply]
Thank you - I'll try that asap

**Comment by b224hisl** [comment]
I'd like to ask about the meaning of using "detailed_route_debug -pa -pa_markers -pin [xx]". 

After executing it, the displayed result is
![image](https://github.com/user-attachments/assets/bf50aafa-4164-4bbf-9bbe-a5efd16baec3)

Accordingly， there are rectangular markers near the pin in the GUI. What does this signify?
![image](https://github.com/user-attachments/assets/545344ab-f17f-4db6-88a0-b35ac7499426)

**Reply by maliberty** [reply]
It is testing placing a via at this location for pin access but finds it is invalid due top the DRC errors shown.  If you click continue you can follow through all the choices it tries.

**Reply by b224hisl** [reply]
thank you～
